tools/token_generator/main.go

```diff
@@ -0,0 +1,51 @@
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+var jwtSecret = []byte(os.Getenv("JWT_SECRET"))
+
+type TokenClaims struct {
+	Version string `json:"version"`
+	jwt.RegisteredClaims
+}
+
+func GenerateJWT() (string, error) {
+	claims := TokenClaims{
+		Version: "0.3.0",
+		RegisteredClaims: jwt.RegisteredClaims{
+			ExpiresAt: jwt.NewNumericDate(time.Now().Add(time.Hour * 24)),
```

Comment:
The (time) duration should be something like a parameter. you could read the value from some ENV variable if set, otherwise set the default (24 hours is maybe to tight..somthing like 30days would be better).

Reply:
Done